log_drains.go

```diff
@@ -42,6 +43,22 @@ func (c *Client) LogDrainsList(app string) ([]LogDrain, error) {
 	return logDrainsRes.Drains, nil
 }
 
+func (c *Client) LogDrainsAddonList(app string, addonID string) (LogDrainsRes, error) {
+	var logDrainsRes LogDrainsRes
+
+	req := &httpclient.APIRequest{
```

Comment:
Most of the time, you shouldn't have to use `httpclient.APIRequest`. Can't you use `c.ScalingoAPI().Subresource*`?

Reply:
Ideally it is more a `SubSubResource` but yes it is entirely possible to concatenate in a parameter